src/sentry/options/manager.py

```diff
@@ -108,6 +108,8 @@ class UnknownOption(KeyError):
 FLAG_AUTOMATOR_MODIFIABLE = 1 << 11
 # Values that are scalar numeric integer values
 FLAG_SCALAR = 1 << 12
+# Values that are meant to be used by the flagpole feature handler
+FLAG_FLAGPOLE_FEATURE = 1 << 13
```

Comment:
Would we use this flag for options that are implicitly registered when features are defined?

Reply:
Yep, that's the idea. I'll be baking this functionality into the feature registration process in getsentry.